### DISCUSSION TITLE
安装微调的依赖文件requirements.txt时报错

### DISCUSSION THREAD

**Discussion by liangzhenbin1992** [zai-org/ChatGLM3#1017] [Q&A] [answered]
![1711125637122](https://github.com/THUDM/ChatGLM3/assets/148993560/89298fa8-5982-4493-a62b-e5778f684e87)
安装微调依赖文件中的mpi4py时报错，先安装了ChatGLM3并能成功运行，在同一个虚拟环境下安装微调文件的依赖时则报错，具体如上。请问有大神知道怎么解决吗？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
mpi 在inux 还有其他apt 包需要装，不是pip就完事了，可以google一下

**Comment by zixiaotan21** [comment]
同样是微调的时候，安装微调的依赖文件requirements.txt时报错，想问问楼主解决了吗
